engine/src/flutter/lib/ui/painting.dart

```diff
@@ -116,19 +116,17 @@ class Color {
   ///
   /// {@template dart.ui.Color.componentsStoredAsFloatingPoint}
   /// > [!NOTE]
-  /// > Each color is stored as normalized floating-point color components,
-  /// > where the final value of each component is approximated by storing
-  /// > `c / 255`, where `c` is one of the four components (alpha, red, green,
-  /// > blue).
+  /// > Each color is stored as floating-point color components, where the final
+  /// > value of each component is approximated by storing `c / 255`, where `c`
+  /// is one of the four components (alpha, red, green, blue).
   /// {@endtemplate}
   const Color(int value)
     : this._fromARGBC(value >> 24, value >> 16, value >> 8, value, ColorSpace.sRGB);
 
-  /// Construct a color with normalized color components from `0.0` to `1.0`.
+  /// Construct a color with color components with a minimum value of `0.0`.
```

Comment:
You can't say minimum value of 0.0 either, the numbers go negative in extended SRGB.